### DISCUSSION TITLE
Trying to Host Tailwind CSS Documentation Locally

### DISCUSSION THREAD

**Discussion by Fearless09** [tailwindlabs/tailwindcss#11161] [Help] [answered]
Hi, Dev

I am trying to host Tailwind CSS locally on my Windows machine. Can anyone guide me on how to do that? Any idea will be welcome and appreciated. Thank you

Happy coding...

**Comment by wongjn** [comment]
As per [the `README.md`](https://github.com/tailwindlabs/tailwindcss.com/blob/master/README.md):

> […] first install the dependencies:
>
> ```bash
> npm install
> ```
>
> Next, run the development server:
>
> ```bash
> npm run dev
> ```

**Reply by wongjn** [reply, marked as answer]
> I have tried it.
> 
> This is the message in the console
> 
> tailwindcss@3.3.2 dev
> 
> > concurrently -n tailwind,oxide-node -c green,yellow 'npm run dev:js' 'npm run dev:rust'
> 
> 'concurrently' is not recognized as an internal or external command, operable program or batch file.

[^1]

> Or is there any step after npm run dev?

[^2]

You seem to be trying to run `npm dev` in this (tailwindlabs/tailwindcss) repository, but the Tailwind CSS documentation is at [tailwindlabs/tailwindcss.com](https://github.com/tailwindlabs/tailwindcss.com) – could you clarify if you were after the documentation site (as the title of this discussion suggests) or something else?

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/11161#discussioncomment-5818164
[^2]: https://github.com/tailwindlabs/tailwindcss/discussions/11161#discussioncomment-5818172

**Reply by Fearless09** [reply]
Okay, what I want is that I am new to tailwind CSS, so I need to download their documentation offline so, that I can go through the documentation without an internet connection. Can you help me out with that

**Reply by Fearless09** [reply]
Wongjn, Thank you very much. I have been able to host it locally on my machine. Can I follow you on twitter?

**Comment by Fearless09** [comment]
I have tried it.

This is the message in the console

 tailwindcss@3.3.2 dev
> concurrently -n tailwind,oxide-node -c green,yellow 'npm run dev:js' 'npm run dev:rust'

'concurrently' is not recognized as an internal or external command,
operable program or batch file.

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/11161#discussioncomment-5818207 to keep the reply chain together.

**Comment by Fearless09** [comment]
Or is there any step after npm run dev?

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/11161#discussioncomment-5818207 to keep the reply chain together.

**Comment by codemodify** [comment]
Is there a way to do it without any nodejs 
no idea why nodejs is taken as a goto "standard"
keep it simple, nodejs is the past 

looking for HTML head links to self hosted
self hosted must be installable without installing nodejs, a simple .css file download form a release page should be sufficient

imagine I'm hosting this in a self contained Go lang app, I need to install half of the planet for a file
